about.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>AC About</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&family=Roboto:wght@300;900&display=swap" rel="stylesheet">
+  <link
+  rel="stylesheet"
+  href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+  integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+  crossorigin="anonymous"
+  /> 
+  <link rel="stylesheet" href="styles/about.css">
+
+</head>
+<body>
+  <header>
+    <div class="nav-top">
+      <ul class="nav-menu-top ">
+        <li class="nav-item-top"><i class="fab fa-facebook-f nav-link-top"></i></li>
+        <li class="nav-item-top"><i class="fab fa-twitter nav-link-top"></i></li>        
+        <li class="nav-item-top"><a href="#" class="nav-link-top">English</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">My Page</a></li>
+        <li class="nav-item-top"><a href="#" class="nav-link-top">Login</a></li>
+      </ul>
+    </div>
+
+    <nav class="navbar">
+      <a href="#" class="logo">AC <span>2022</span></a>
+      <ul class="nav-menu ">
+        <li class="nav-item"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+        <li class="nav-item"><a href="#" class="nav-link">AC Campaign</a></li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="hero">
+      <h2>"Hello World! We Are Algerian Coders</h2>
+      <h1>البرمجة الجزائرية Algerian Code 2022</h1>
+      <div><p>Hello and welcome to the blog!
+
+        This blog is dedicated to coders all around the globe to help them in their journey in the world of software developement.
+        
+        If you want to come along with me to improve our skills throughout different projects and problems that I face in my job.
+        We will learn how to approach and solve problems, how to debug code and find the source of the problem…
+        
+        If you are a new coder or someone who wants to keep learning, than this is the place for you!
+       </p></div>
+      <p>
+        Please contact us if you have any question:<br><b>sample-email@sample.mail</b>
+      </p>
+    </section>
```

Comment:
- [ ] @Mehdi-Rh , great job for such a wonderful design, you have just a little change to make to the design to give an inch-perfect work. kindly make your `<section id="hero">` to have a dark background and the paragraph that contains the text `Hello and welcome to the blog!` to have a white background just as it looks in the design
     your design 👇 
                       ![image](https://user-images.githubusercontent.com/66097371/153713391-3e9e4174-9b0f-4b3a-b263-01f7e9c58aa3.png)

   Figma design 👇 
             
![image](https://user-images.githubusercontent.com/66097371/153713416-60da00d1-1448-467b-ab0b-2e8631b83d45.png)